static/css/style.css

```diff
@@ -101,18 +118,18 @@ code {
 	text-align: center;
 }
 /* about page*/
-img{
+img {
```

Comment:
As we agreed last time, please remove this definition of img element  from style.css and use class or id instead. 
The definition 100px of image element is overwriting the responsiveness of all images in allover the site.
Avoid restyling basic html elements for specific or sporadic needs.